### PR TITLE
BrowserStack Examples WebdriverIO Release 1.0

### PR DESCRIPTION
**Pull request by swapnsharma**
BrowserStack Examples WebdriverIO Release 1.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,366 @@
...
+
+This BrowserStack Example repository demonstrates a WebdriverIO tests framework written in Mocha and nodeJS with parallel testing capabilities. The WebdriverIO test scripts are written for the open source [BrowserStack Demo web application](https://bstackdemo.com) ([Github](https://github.com/browserstack/browserstack-demo-app)). This BrowserStack Demo App is an e-commerce web application which showcases multiple real-world user scenarios. The app is bundled with offers data, orders data and products data that contains everything you need to start using the app and run tests out-of-the-box.
+
+The WebDriverIO tests are run on different platforms like on-prem, docker and BrowserStack using various run configurations and test capabilities.
+
+---
+
+## Repository setup
+
+- Clone the repository
```
[the remark this answers is not in the document]
Comment (mudassarsyed) [reply README.md:17]: this has been done

File: src/test/suites/user/image_not_loading_user.spec.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    $('#username input').setValue(browser.config.accounts[2].username + '\n');
+    $('#password input').setValue(browser.config.accounts[2].password + '\n');
+    $('#login-btn').click();
+    expect($('.username')).toHaveText('image_not_loading_user');
+
+    all_images = $$("div.shelf-item__thumb img").map(function(element){
+      return element.getAttribute("src")
+    });
+
+    expectChai(_.every(all_images,  function (value) {
```
[the remark this answers is not in the document]
Comment (mudassarsyed) [reply src/test/suites/user/image_not_loading_user.spec.js:25]: resolved this

File: README.md [hunk 3]
```diff
@@ -1,2 +1,366 @@
...
+## Running Your Tests
+
+### Run a specific test on your own machine
+
+- How to run the test?
+
+  To run the default test scenario (e.g. End to End Scenario) on your own machine, use the following command:
+  
+  ```sh
+  npm run test
```
[the remark this answers is not in the document]
Comment (mudassarsyed) [reply README.md]: corrected

File: README.md [hunk 4]
```diff
@@ -1,2 +1,366 @@
-# browserstack-examples-webdriverio
-This repository showcases the BrowserStack integration features with the WebdriverIO using the BrowserStack sample demo canonical app. More details:https://browserstack.atlassian.net/wiki/spaces/CE/pages/2091057195/Customer+Demo+Repositories
+![Logo](https://www.browserstack.com/images/static/header-logo.jpg)
+
+# BrowserStack Examples WendriverIO <a href="https://webdriver.io/"><img src="https://avatars.githubusercontent.com/u/72550141?s=48&v=4" alt="WebdriverIO" height="22" /></a> <a href="https://nodejs.org/en/"><img src="https://brandslogos.com/wp-content/uploads/images/large/nodejs-icon-logo.png" alt="nodejs" height="22" /></a> <a href="https://mochajs.org/"><img src="https://brandslogos.com/wp-content/uploads/images/large/mocha-logo.png" alt="mochs" height="22" /></a>
```
[the remark this answers is not in the document]
Comment (mudassarsyed) [reply README.md]: This has been corrected

File: src/test/suites/product/vendor.filter.spec.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+  })
+
+  it('Apply vendor filter', () => {
+    $("input[value='Apple'] + span").click();
+    $("input[value='Samsung'] + span").click();
+    browser.pause(5000)                                               // Example for static wait
+    all_phones = $$(".shelf-item__title").map(function(element){
+      return element.getText()
+    });
+    expectChai(_.every(all_phones,  function (value) {
```
[the remark this answers is not in the document]
Comment (mudassarsyed) [reply src/test/suites/product/vendor.filter.spec.js:20]: Can u recheck on this

**Review by mudassarsyed — CHANGES_REQUESTED** [review]

File: src/test/suites/product/vendor.filter.spec.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (swapnsharma) [reply src/test/suites/product/vendor.filter.spec.js:20]: already updated, please take the latest and check.

[the remark this answers is not in the document]
Comment (mudassarsyed) [reply src/test/suites/product/vendor.filter.spec.js:20]: ok corrected

**Pull request merged**